src/controller/WebController.php

```diff
@@ -233,7 +215,8 @@ public function invokeVocabularyConcept(Request $request)
     public function invokeFeedbackForm($request)
     {
         $template = $this->twig->loadTemplate('feedback.twig');
-        $this->setLanguageProperties($request->getLang());
+        //$this->setLanguageProperties($request->getLang());
```

Comment:
Remove instead of commenting out.

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764